### PR TITLE
[WR-454] Get Realm Identity count API

### PR DESCRIPTION
**Pull request by Mrhea**
Draft until tested

### CODE REVIEW

File: identityClient/api.go [hunk 1]
```diff
@@ -1374,3 +1374,13 @@ type ListIdentitiesMFADeviceResponse struct {
...
 }
+
+// CountRealmIdentitiesResponse wraps the count of Identities in a Realm
+type CountRealmIdentitiesResponse struct {
+	IdentityCount int `json:"identity_count"`
+}
+
+// CountRealmIdentitiesRequest wraps the request to get the number of Identities in a Realm
+type CountRealmIdentitiesRequest struct {
+	RealmName string `json:"realm_name"`
```
Comment (rosasck) [inline identityClient/api.go:1385]: since its a path var u dont need the json tag!

File: identityClient/identityClient.go [hunk 2]
```diff
@@ -1449,3 +1449,14 @@ func (c *E3dbIdentityClient) DeleteMFAInformation(ctx context.Context, params De
 	return err
 
 }
+
+func (c *E3dbIdentityClient) GetRealmUserCount(ctx context.Context, params CountRealmIdentitiesRequest) (CountRealmIdentitiesResponse, error) {
```
Comment (rosasck) [inline identityClient/identityClient.go:1453]: comment

File: keycloakClient/keycloakClient.go [hunk 3]
```diff
@@ -1178,6 +1178,12 @@ func (c *Client) RemoveMFADevice(accessToken, realmName, userID, mfaID string) e
 	return err
 }
 
+func (c *Client) GetRealmUserCount(accessToken, realmName string) (CountRealmIdentitiesResponse, error) {
```
Comment (rosasck) [inline keycloakClient/keycloakClient.go:1181]: comment

**Review by rosasck — APPROVED** [review]
pending changes

File: identityClient/api.go (hunk 1 above)
Comment (Mrhea) [reply identityClient/api.go:1385]: This has been in a draft for about .5 seconds, so fast 🏃‍♀️ 🏃‍♀️

**Pull request closed without merging**